articles/multifactor-authentication/mfa-from-id-token.md

```diff
@@ -0,0 +1,28 @@
+---
+description: Describes how to determine if a user has done multifactor authentication using their id_token and JWT
+---
+
+## Determine if a user has performed multifactor authentication
+
+At times, it's necessary to determine if a particular user has had the additional security of multifactor authentication applied to their session. For instance, a user may be allowed access to sensitive data or allowed to reset their password only after further confirming their identity using MFA.
+
+To determine this for a particular user session, developers can check the claims information available on the `id_token`. 
+The `id_token` contains claims information relevant to the usage of multifactor authentication. After retrieving the `id_token`, [as shown here](tokens/id_token), the value of the `amr` claim can be evaluated.
```

Comment:
I'd say that the value might not be present or might contain a different values other than just `'mfa'`, so you should always check that the `'mfa'` string is in the array and not just the trusting that the existence of the array is enough to tell that. Right now the only value is mfa, but will change for sure in future.